doc/source/conf.py

```diff
@@ -173,6 +173,26 @@ def add_ipython_time(app, docname, source):
     source[0] = "\n".join(modified_lines)
     # logger.info(source[0])
 
+def adjust_image_path(app, docname, source):
+    """Adjust the HTML label used to insert images in our examples.
```

Comment:
```suggestion
    """Adjust the HTML label used to insert images in the examples.
```
Avoid "our" and "we".